packages/api/src/beacon/server/proof.ts

```diff
@@ -3,41 +3,6 @@ import {ChainForkConfig} from "@lodestar/config";
 import {ApplicationMethods, FastifyRoutes, createFastifyRoutes} from "../../utils/server/index.js";
 import {Endpoints, getDefinitions} from "../routes/proof.js";
 
-// TODO: revisit, do we need still need to override handlers?
-
 export function getRoutes(config: ChainForkConfig, methods: ApplicationMethods<Endpoints>): FastifyRoutes<Endpoints> {
   return createFastifyRoutes(getDefinitions(config), methods);
-  // const serverRoutes = createFastifyRoutes(definitions, methods);
-
-  // return {
-  //   // Non-JSON routes. Return binary
-  //   getStateProof: {
-  //     ...serverRoutes.getStateProof,
-  //     handler: async (req) => {
-  //       const args = definitions.getStateProof.req.parseReq(req);
```

Comment:
we could implement the same logic in `serialize` / `deserialize` in the route without overriding the handler if that's the format we wanna keep